modules/migration-error-messages.adoc

```diff
@@ -12,37 +12,42 @@ This section describes common error messages you might encounter with the {mtc-f
 [id="ca-certificate-error-in-console_{context}"]
 == CA certificate error in the {mtc-short} console
 
-If a `CA certificate error` message is displayed the first time you try to access the {mtc-short} console, the likely cause is the use of self-signed CA certificates in one of the clusters.
+If the {mtc-short} console displays a `CA certificate error` message the first time you try to access it, the likely cause is that a cluster uses self-signed CA certificates.
 
-To resolve this issue, navigate to the `oauth-authorization-server` URL displayed in the error message and accept the certificate. To resolve this issue permanently, add the certificate to the trust store of your web browser.
+To resolve this issue, navigate to the `oauth-authorization-server` URL in the error message and accept the certificate.
```

Comment:
Just wondering if you should add "temporarily" or "for this session", since the next sentence says about resolving it "permanently"?